package.json

```diff
@@ -78,7 +78,7 @@
     "eslint-plugin-react": "^7.31.8",
     "eslint-plugin-react-hooks": "^4.6.0",
     "eslint-plugin-testing-library": "^6.2.0",
-    "jsdom": "^22.1.0",
+    "jsdom": "^25.0.1",
```

Comment:
Update to be able to use `HTMLElement#ariaDisabled` in specs.

https://github.com/jsdom/jsdom/pull/3655